src/mmw/js/src/account/views.js

```diff
@@ -48,6 +50,34 @@ var LinkedAccountsView = Marionette.ItemView.extend({
             // Fetch user again to save new HydroShare Access state
             self.model.fetch();
         });
+    },
+
+    unlinkHydroShare: function() {
+        var self = this,
+            confirm = new modalViews.ConfirmLargeView({
+                model: new modalModels.ConfirmModel({
+                    titleText: 'Remove link',
+                    question: [
+                        'This will unlink your MMW account from HydroShare. ' +
+                        'Any currently syncing projects will disconnect from ' +
+                        'their HydroShare resources. Any resources already ' +
+                        'exported to HydroShare will remain there.',
+
+                        'If you re-link to HydroShare, project export will ' +
+                        'have to be re-enabled. They will export new resources.',
+
+                        'Continue?'
+                    ],
+                    confirmLabel: 'Remove link',
+                })
+            });
+
+        confirm.render();
+        confirm.on('confirmation', function() {
+            $.post('/user/hydroshare/logout/').done(function() {
```

Comment:
Do we need to worry about reporting failure here? I used "offline" mode to test this and it fails relatively gracefully by just not doing anything. We may want to consider showing some failure message as a way of confirming that we did try to unlink the account.